preprocessing/feature_selection_MoranI/spatially_variable_genes_moransI.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+
+# Author_and_contribution: Niklas Mueller-Boetticher; created script
+# Author_and_contribution: Qirong Mao; Implemented feature selection with Moran's I
+
+import argparse
+
+# TODO adjust description
+parser = argparse.ArgumentParser(description="SVG selection using Squidpy (Moran's I)")
+
+parser.add_argument(
+    "-c", "--coordinates", help="Path to coordinates (as tsv).", required=True
+)
+parser.add_argument(
+    "-m", "--matrix", help="Path to (transformed) counts (as mtx).", required=True
+)
+parser.add_argument(
+    "-f", "--features", help="Path to features (as tsv).", required=True
+)
+parser.add_argument(
+    "-sc", "--spatial_connectivities", help="Path to spatial connectivities (as mtx).", required=True
```

Comment:
rename the arg to be consistent with prior nomenclature

`parser.add_argument(
    "-n",
    "--neighbors",
    help="Path to neighbor definitions. Square matrix (not necessarily symmetric) where each row contains the neighbors of this observation (as mtx).",
    required=False
)`